libs/sdk-ui-tests-e2e/cypress/support/constants.ts

```diff
@@ -0,0 +1,11 @@
+// (C) 2021 GoodData Corporation
+
+export const getHost = () => Cypress.env("HOST");
+
+export const getProjectId = () => Cypress.env("WORKSPACE");
+
+export const getUsername = () => Cypress.env("USERNAME");
+
+export const getPassword = () => Cypress.env("PASSWORD");
+
+export const getDashboardsUrl = (projectId: string) => `${getHost()}/dashboards/#/project/${projectId}`;
```

Comment:
This will most likely not be needed in our case.

Reply:
Well, it's not needed while testing against the playground. May be this will be refactored in the future

Reply:
It will never be used :) this is KD specific URI which will not be used by the dashboard components